modules/local/cat_fasta.nf

```diff
@@ -0,0 +1,27 @@
+process CAT_FASTA {
+    tag "$ref_genome_fasta"
+    label 'process_low'
+
+    conda "conda-forge::sed=4.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
+
+    input:
+    path ref_genome_fasta
+    path virus_db_fasta
+
+    output:
+    path '*_plus_viraldb.fasta', emit: plus_fasta
+    path "versions.yml"        , emit: versions
+
+    script:
+    """
+    cat $ref_genome_fasta $virus_db_fasta > ${ref_genome_fasta.simpleName}_plus_viraldb.fasta
```

Comment:
`viraldb` is hardcoded, don't you want `virus_db_fasta.simpleName`?

Reply:
Nope, the viraldb would stay the same unless the user provides a custom fasta with sequences. So this would change between `hg38` and `hg19` but the viraldb would be the same.

Open to better naming ideas as well. I believe this was an external script in CTAT-vif that you ran before running the pipeline, but we can handle it better.

Reply:
Good enough for me now